docs/src/guidelines.md

```diff
@@ -80,6 +80,22 @@ The following is a non-exhaustive list of expected behaviour for methods.
 - Vector inputs should have the correct size. If necessary, the user should pass them using views or slices.
 - The triplet format does not assume order nor uniqueness.
 
+## [Show](@id show)
+
+To further specialize your model, you can also define `show_header` and possibly `show`.
+The default `show_header` simply prints the `typeof` the NLPModel, so it should be specialized with the specific information that you prefer. For instance, `SlackModel` defines
+```julia
+show_header(io :: IO, nlp :: SlackModel) = println(io, "SlackModel - Model with slack variables")
+```
+Furthermore, we define a general `show` that calls `show_header` and specific `show` functions for the `meta` and the `counters`. If your model does not have `counters` in the default location, you must define `show` as well. Alternatively, you may desire to change the behaviour of show. Here is an example, again from `SlackModel`:
```

Comment:
... you must define `show` **for them** as well

Reply:
Done